snippets/rust/rust.json

```diff
@@ -285,6 +300,26 @@
         "body": ["if ${1:condition} {", "    ${2:todo!();}", "}"],
         "description": "if … { … }"
     },
+    "ifelse": {
+        "prefix": "ifelse",
+        "body": [
+            "if ${1:condition} {",
+            "    ${2:todo!();}",
```

Comment:
nit: remove this semicolon after `todo!()` so that it doesn't return `()` and it infers the other branch.